src/ui/action/handler/antares-study/area/create.cpp

```diff
@@ -296,7 +296,7 @@ void Create::createActionsForAStandardAreaCopy(Context& ctx, bool copyPosition)
         auto end = area->thermal.list.end();
```

Comment:
Change into a `for(auto& cluster : area->thermal.list)`, quickly done, and more clear